Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -63,6 +68,17 @@ def setUp(self):
         self.assertNotEqual(
             self.subscription_id, "", "subscription_id is empty"
         )
+        self.assertNotEqual(self.container_name, "", "container_name is empty")
+        self.assertNotEqual(
+            self.storage_account,
+            "",
+            "No storage account found in the specified resource group.",
+        )
+        self.assertNotEqual(
```

Comment:
Will this assert work as expected if storage_account or storage_account_key is 'None'?

Reply:
just added two more assert, Now, the test will fail if self.storage_account is either None or an empty string.

Reply:
done